person.rb

```diff
@@ -9,6 +10,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @age = age
     @parent_permission = parent_permission
     @corrector = Corrector.new
+    @books = []
```

Comment:
- [ ] This class attr should be rentals not books, kindly refer to the previous [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars) if you need more insight. Also, please don't forget to update the `attr_accessor` as well.

Reply:
thank you very much for your kind reply I forgot to see this 
![image](https://user-images.githubusercontent.com/83329154/146474822-1dc0b2ac-202d-435d-a3da-59139fd572e9.png)
when I was creating the variables' names I will change it to rentals as well as for books.rb I will change it to rentals too

Reply:
Hey @omar-labana I pushed the changes is that it ?